examples/compass.py

```diff
@@ -0,0 +1,21 @@
+from microbit import *
+
+needles = (Image.CLOCK12, Image.CLOCK1, Image.CLOCK2, Image.CLOCK3,
+           Image.CLOCK4, Image.CLOCK5, Image.CLOCK6, Image.CLOCK7, 
+           Image.CLOCK8, Image.CLOCK9, Image.CLOCK10, Image.CLOCK11)
+
+#You need to spin the microbit about a few times to help calibrate the compass
+compass.calibrate()
+
+#Spin needle until calibrated.
+while compass.is_calibrating():
+    for i in range(12):
+        display.print(needles[i])
+        sleep(80)
+display.clear()
+
+#Try to keep needle pointed in (roughly) the same direction
+while True:
+    sleep(500)
+    needle = ((15-compass.heading())//30)%12
```

Comment:
Since new python users might use this example, it might be good to be more pythonic on the line spacing:
``needle = ((15 - compass.heading()) // 30) % 12``

Also on the comments, add a space after the #